src/e2e-test/features/bigqueryexecute/BQExecute.feature

```diff
@@ -173,3 +173,44 @@ Feature: BigQueryExecute - Verify data transfer using BigQuery Execute plugin
     Then Open and capture logs
     Then Verify the pipeline status is "Succeeded"
     Then Verify 1 records updated in BigQuery table: "bqSourceTable" with query "bqExecuteCountDMLUpdate"
+
+  @BQEXECUTE_SOURCE_TEST @BQ_SINK_TEST @BQ_EXECUTE_SQL @BQExecute_Required
+  Scenario: Verify the successful records transfer from Bigquery(source) to Bigquery(sink) using BQExecute with allow relaxation schema as false.
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Conditions and Actions"
+    When Select plugin: "BigQuery Execute" from the plugins list as: "Conditions and Actions"
+    When Navigate to the properties page of plugin: "BigQuery Execute"
+    Then Replace input plugin property: "projectId" with value: "projectId"
+    Then Enter textarea plugin property: "sql" with value: "bqExecuteQuery"
```

Comment:
Don't use select query, instead use the Insert sql query to insert the record through BQ execute.

Reply:
insert query used